std/assembly/polyfills.ts

```diff
@@ -1,12 +1,12 @@
 export function bswap<T extends number>(value: T): T {
   if (isInteger<T>()) {
     if (sizeof<T>() == 2) {
-      return <T>((value << 8) | ((value >> 8) & <T>0x00FF));
+      return <T>((value << 8) | ((value >>> 8) & <T>0x00FF));
```

Comment:
Was this wrong previously?

Reply:
no, both return same result. But now more consistent